subhub/tests/unit/stripe/customer/test_stripe_customer.py

```diff
@@ -101,64 +107,55 @@ def test_stripe_webhook_customer_source_expiring(mocker):
 
 def test_stripe_webhook_customer_subscription_created(mocker):
     data = {
-        "event_id": "evt_00000000000000",
-        "event_type": "customer.subscription.created",
-        "customer_id": "cus_00000000000000",
-        "subscription_id": "sub_00000000000000",
-        "current_period_start": 1519435009,
-        "current_period_end": 1521854209,
-        "canceled_at": 1519680008,
-        "days_until_due": None,
-        "default_payment_method": None,
-        "plan_id": "subhub-plan-api_00000000000000",
-        "plan_amount": 500,
-        "plan_currency": "usd",
-        "plan_interval": "month",
-        "status": "canceled",
-        "trial_start": None,
-        "trial_end": None,
-        "tax_percent": None,
-        "application_fee_percent": None,
-        "user_id": None,
+        "uid": "tester123",
```

Comment:
food for thought the tests could use dict() + keyword instead of {} and 'quoted-strings' just like the app code does now...

Reply:
I implemented this on this function, we can update the others in a future pass.